types/types.d.ts

```diff
@@ -80,6 +79,8 @@ interface Signer {
 }
 
 declare namespace fastifySession {
+  export type CallbackSession = (err: null | Error, result: undefined | Fastify.Session) => void
```

Comment:
Wouldnt this be better?
```suggestion
  export type CallbackSession = ((err: Error, result: undefined) => void) | ((err: null, result: undefined | Fastify.Session) => void)
```

Reply:
```suggestion
  export type CallbackSession = ((err: Error) => void) | ((err: null, result: Fastify.Session) => void)
```

Reply:
@Uzlopak - I think what @climba03003 suggested is best. We don't need to return an undefined result, we just don't return anything at all, which makes `result` undefined.

Reply:
@Uzlopak - I think what @climba03003 suggested is best. We don't need to return an undefined result, we just don't return anything at all, which makes `result` undefined.